src/components/common/DocumentsDelivery.vue

```diff
@@ -109,6 +113,14 @@ export default class DocumentsDelivery extends Mixins(CommonMixin) {
     return (this.validate && !this.getFlagsReviewCertify.isValidDocumentOptionalEmail)
   }
 
+  get userEmail (): string {
+    return this.applicantEmail ? this.applicantEmail : this.getUserEmail
+  }
```

Comment:
compute email value. It is different for other filings(user email) and Restorations (applicant email)

Reply:
Looks good.

No need to change but here's a shortcut: `return (this.applicantEmail || this.getUserEmail)`

Reply:
I agree! I will rename the prop.

Reply:
LGTM.